llpc/test/shaderdb/OOB_Check_Load_Array_lit.frag

```diff
@@ -0,0 +1,25 @@
+#version 450 core
+
+layout (location = 0) in vec4 inColor;
+layout (location = 0) out vec4 outFragColor;
+layout (binding = 0) uniform Constant { int array_index; } c;
+
+void main() {
+  vec4 data[2048][2];
+  data[c.array_index][c.array_index] = inColor.rrrr;
+
+  outFragColor = inColor * data[c.array_index + 2048][c.array_index + 2048];
+}
+
+// BEGIN_SHADERTEST
+/*
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v %gfxip %s -scratch-bounds-checks | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL: {{^// LLPC}} SPIRV-to-LLVM translation results
+; SHADERTEST: %[[B:[0-9]+]] = icmp ult i32 %[[A:[0-9]+]], 2048
```

Comment:
Would it be possible to also match the array? The way these checks are written, it's not immediately obvious how this array is lowered to LLVM.